vllm/v1/core/kv_cache_utils.py

```diff
@@ -684,13 +769,16 @@ def unify_hybrid_kv_cache_specs(kv_cache_spec: dict[str, KVCacheSpec]):
                     sliding_window=spec.sliding_window,
                 )
 
+    if not is_hybrid(kv_cache_spec):
```

Comment:
it should be 
```if is_hybrid(kv_cache_spec): raise error```? 
since the goal is  make sure kv_cache_spec only have one type

Reply:
Thanks for catching up this problem. Will fix it later.